inference-engine/thirdparty/clDNN/src/network.cpp

```diff
@@ -218,7 +220,8 @@ network::network(program::ptr program, stream::ptr stream, bool is_internal, boo
     , _memory_pool(new memory_pool(program->get_engine()))
     , _internal(is_internal)
     , _is_primary_stream(is_primary_stream)
-    , _reset_arguments(true) {
+    , _reset_arguments(true)
+    , _num_executions(0) {
```

Comment:
why should we need this in network's api? cant it be just in the debug_config?

Reply:
I remove _num_executions because it is no more needed. When dump mode is on, only one inference running is allowed by latest commit.